src/app/home/components/DataList/Plans/types.ts

```diff
@@ -0,0 +1,15 @@
+import { IMigMigration } from '../../../../../client/resources/conversions';
+
+export interface IMigration extends IMigMigration {
```

Comment:
This makes perfect sense since tableStatus is extended in the selectors, and the extended version is what we pass around. I wonder if we should eventually make explicit interfaces for all of those types in `client/resources/conversions` though, I'm not sure how much the `ReturnType<typeof ...>` does for us (at least in some cases). I suppose since the params of `createMigMigration` have types and the returned object just reuses those directly, it's still useful. Might be worth looking at later.

Reply:
@mturley, thanks for your review. 
I changed the interface name to `IMigrationWithStatus` it makes sense.

I was wondering about `ReturnType<typeof ...>` too. It seems handy but not as flex as formal interfaces. I'll leave it for another PR.